eras/babbage/test-suite/test/Test/Cardano/Ledger/Babbage/Serialisation/Tripping.hs

```diff
@@ -10,7 +10,7 @@ import Cardano.Ledger.Babbage (BabbageEra)
 import Cardano.Ledger.Babbage.Rules (BabbageUtxoPredFailure)
 import Cardano.Ledger.Block (Block)
 import Cardano.Ledger.Core
-import Cardano.Ledger.Crypto (StandardCrypto)
+import Cardano.Protocol.Crypto (StandardCrypto)
```

Comment:
This looks like it requires 
`cardano-protocol-tpraos >=1.4`  in cardano-ledger-babbage-test.cabal

Reply:
Done. Thank you for identifying it!